datamol/mol.py

```diff
@@ -1376,3 +1384,42 @@ def get_atom_positions(
         positions = positions[mapped_indices, :]
 
     return positions
+
+
+def remove_salts(mol: Mol, remover: Optional[SaltRemover] = SALT_REMOVER) -> Mol:
```

Comment:
I don't know what you think but I would remove the `remover` arg. Given how simple the function is, the user will just make its own function if he needs something custom.